AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -293,6 +282,27 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
 print("Using %s as the reference antenna "%(ant_list[0]))
 h5.select(compscans='interferometric_pointing',ants=ant_list)
 
+if len(opts.channel_mask)>0:
+    pickle_file = open(opts.channel_mask , "rb" )
+    channel_flags = pickle.load(pickle_file)
+    pickle_file.close()
+    NC = len(h5.channels)
```

Comment:
While this script suffers from a slew of PEP8 deficiencies, there seems to be general adherence to at least one rule: lower-case variable names. It will help a lot to preserve the little bit of homogeneity we have 😃